lib/Serialization/Serialization.cpp

```diff
@@ -4525,7 +4525,7 @@ void swift::serialize(ModuleOrSourceFile DC,
 
   bool hadError = withOutputFile(getContext(DC), options.OutputPath,
                                  [&](raw_ostream &out) {
-    SharedTimer timer("Serialization (swiftmodule)");
+    SharedTimer timer("Serialization, swiftmodule");
```

Comment:
Bizarre. Okay, I'll assume that was intentional. Is that at least in a doc comment somewhere? :-/